packages/common/src/store/wallet/utils.ts

```diff
@@ -1,48 +1,43 @@
+import { AUDIO, FixedDecimal, AudioWei } from '@audius/fixed-decimal'
 import { PublicKey } from '@solana/web3.js'
 
 import { useUser } from '~/api/tan-query/users/useUser'
 import { Maybe, Nullable } from '~/utils'
-import { stringAudioToBN, stringWeiToAudioBN } from '~/utils/wallet'
 
 import { BadgeTier } from '../../models/BadgeTier'
 import { ID } from '../../models/Identifiers'
-import {
-  BNAudio,
-  SolanaWalletAddress,
-  StringAudio,
-  StringWei
-} from '../../models/Wallet'
+import { SolanaWalletAddress, StringWei } from '../../models/Wallet'
 
 export type BadgeTierInfo = {
   tier: BadgeTier
-  minAudio: BNAudio
+  minAudio: FixedDecimal<AudioWei>
```

Comment:
Not opposed to doing this but I think what we've been doing is branded BigInts and using fixed decimal when needing to do formatting things